libs/java/server_common/src/test/java/com/yahoo/athenz/common/server/store/AWSCredentialsRefresherTest.java

```diff
@@ -0,0 +1,119 @@
+/*
+ *  Copyright 2020 Verizon Media
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package com.yahoo.athenz.common.server.store;
+
+import com.amazonaws.auth.AWSCredentials;
+import com.yahoo.athenz.zms.ResourceException;
+import org.eclipse.jetty.client.HttpClient;
+import org.eclipse.jetty.client.api.ContentResponse;
+import org.mockito.Mockito;
+import org.testng.annotations.Test;
+
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeoutException;
+
+import static com.yahoo.athenz.common.server.store.AWSInstanceMetadataFetcherTest.AWS_IAM_ROLE_INFO;
+import static com.yahoo.athenz.common.server.store.AWSInstanceMetadataFetcherTest.AWS_INSTANCE_DOCUMENT;
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.fail;
+
+public class AWSCredentialsRefresherTest {
+    @Test
+    public void testAWSCredentialsUpdaterExceptions () {
+
```

Comment:
Extracted tests from CloudStoreTest covering the new classes (AWSCredentialsRefresherTes, AWSInstanceMetadataFetcherTest and S3ClientFactoryTest)